boards/google,kevin

```diff
@@ -24,19 +24,16 @@ assert_driver_present cros-ec-rtc-driver-present cros-ec-rtc
 assert_device_present cros-ec-rtc-probed cros-ec-rtc cros-ec-rtc.*
 
 assert_driver_present cros-ec-sensors-driver-present cros-ec-sensors
-if kernel_greater_than "5.5"; then
-    assert_device_present cros-ec-sensors-accel0-probed cros-ec-sensors cros-ec-accel.11.*
-    assert_device_present cros-ec-sensors-accel1-probed cros-ec-sensors cros-ec-accel.13.*
-    assert_device_present cros-ec-sensors-gyro0-probed cros-ec-sensors cros-ec-gyro.12.*
 
+if kernel_greater_than "5.5"; then
     assert_driver_present cros-ec-sensorhub-driver-present cros-ec-sensorhub
-    assert_device_present cros-ec-sensorhub-probed cros-ec-sensorhub cros-ec-sensorhub.1.*
-else
-    assert_device_present cros-ec-sensors-accel0-probed cros-ec-sensors cros-ec-accel.0
-    assert_device_present cros-ec-sensors-accel1-probed cros-ec-sensors cros-ec-accel.1
-    assert_device_present cros-ec-sensors-gyro0-probed cros-ec-sensors cros-ec-gyro.0
+    assert_device_present cros-ec-sensorhub-probed cros-ec-sensorhub cros-ec-sensorhub.*
 fi
 
+assert_device_present cros-ec-sensors-accel0-probed cros-ec-sensors cros-ec-accel.*
```

Comment:
The issue with this is that there are multiple (2) files that will match the `cros-ec-accel.*` glob. So `assert_device_present` will see each of them as a different parameter. The first will be used as it's supposed to, but the second will be passed to the `timeout` command, which I think will cause an error. So the `*` globbing can only be used in the bootrr commands when it matches a single file (which is why I couldn't have #40 be a single check).

Reply:
@hardboprobot sounds sensible to me, the only thing I'm not sure about is requiring `bash` instead of `sh`. But if you could create a PR we can discuss/test it further there.